### ISSUE TITLE
ProtoGameState is null in GetClosestXTask

### ISSUE THREAD

**Issue by ghost** [Protoalliance/vindinium-client#2]
It seems that the ProtoGameState variable is null in GetClosestXTasks, e.g. GetClosestBotTask and GetClosestMineTask.

Since the bb is passed in when the task is constructed and the gameState is set in MineBot, we need to investigate why this is occurring.

**Comment by meneal** [comment]
Not sure why this was happening since I didn't get the push on your bot.  I
have a working bot in the system now.  Here's a link to his poor game play:
http://vindinium.org/wcojoqjh.

**Comment by meneal** [comment]
This last post was a false positive.  I fixed it though in my last commit 3a6b2ad5ada54385c801b2fdf58d60a2d6b2c476.  Closing.   Let me know if what I did somehow doesn't fix minebot.

**Comment by meneal** [comment]
Just checked Minebot and it's still broken.  Try changing around the actual MineBot class to reflect the way that I implemented BloodthirstBot and see if that takes care of it.

**Comment by meneal** [comment]
I think I fixed it in 927ab6c370057ebe2df3fe82e8c68d2a87dad803.  I changed line 58 in GetClosestMineTask to:
 `state = bb.getGameState()`  

You'd want that anyway since otherwise you'd only get the game state when the object is constructed, you definitely would want to update it every time it runs.  I think this may be from my old code and I fixed it later.  

The game at least runs minebot now, but it looks like it's making the game result in a draw, which I guess means that we aren't responding fast enough.  Hopefully this helps though, I know that at least bloodthirstbot is finishing execution.  Those Game URL links are actually live, that's how I know that we end in a draw.
